From fc08c1d514514ef027ff28f74fe815e60d63f491 Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Tue, 12 Jul 2016 12:50:34 +0300
Subject: [PATCH 0629/2241] mmc: dts: a70x0: disable eMMC high-speed DDR
 mode(1.8V I/O)

eMMC high-speed DDR mode(1.8V I/O) will not work in Armada-70x0-DB board
due to HW limitation, this patch disable the eMMC high-speed DDR mode
and move the flag to board specific device tree.

Change-Id: I11f7aab9e50d35f1d699ff28b1106792ba726dc1
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31064
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Victor Gu <xigu@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db.dtsi | 1 +
 arch/arm64/boot/dts/marvell/armada-ap806.dtsi   | 3 ---
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi b/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
index 8485b62..421711f 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
@@ -68,6 +68,7 @@
 			};
 			sdhci@6e0000 {
 				bus-width = <4>;
+				no-1-8-v;
 				non-removable; /* For now, need to configure the GPIO for card detect */
 			};
 		};
diff --git a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
index 62bb261..9138af9 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
@@ -275,9 +275,6 @@
 				compatible = "marvell,xenon-sdhci";
 				reg = <0x6E0000 0x300>;
 				interrupts = <GIC_SPI 16 IRQ_TYPE_LEVEL_HIGH>;
-				mmc-ddr-1_8v;
-				mmc-hs200-1_8v;
-				mmc-hs400-1_8v;
 				xenon,slotno = <0>;
 				status = "disabled";
 			};
-- 
2.7.4

